examples/steel/OWGCounterInv.fst

```diff
@@ -53,7 +54,10 @@ let ghost_gather (#uses:inames) (v1 #v2:G.erased int) (r:ghost_ref int)
       (fun _ -> True)
       (fun _ _ _ -> v1 == v2)
   = ghost_gather_pt #_ #_ #_ #_ #v1 #v2 r;
-    ()
+    rewrite_slprop
+      (ghost_pts_to _ (sum_perm (half_perm full_perm) (half_perm full_perm)) _)
+      (ghost_pts_to _ full_perm _)
+      (fun _ -> ())
```

Comment:
Surprising ... any explanation for why this needed a change?

Reply:
Didn't debug more, surprising that the failure is in permissions rewriting. FWIW, I checked that there are no Zeta regressions with this branch.